### DISCUSSION TITLE
dragonchess

### DISCUSSION THREAD

**Discussion by GM-ILLESCAS** [fairy-stockfish/Fairy-Stockfish#765] [Variants]
I have a problem to test dragonchess on Fairy-Stockfish. I loose on time before even starting the game. I would very much appreciatte some help. Thanks! GM Miguel Illescas

**Comment by ianfab** [comment]
Hi Miguel, could you please elaborate a bit, i.e., what are the exact steps to reproduce the issue?

**Comment by GM-ILLESCAS** [comment]
Hi Fabian,
I tried again and it works well, only problem if I try to set up time
control
https://vimeo.com/926174781/cca5e89899?share=copy

Is it possible to analyse a game which I played?

By the way, I would like to give you my dragon piece design, would you
include it?

I would love to promote the dragonchess, organizing several tournaments
with top players, I wrote and got san answer :-)
https://github.com/gbtami/pychess-variants/issues/1338

Thanks Fabian!
Miguel Illescas

**Reply by ianfab** [reply]
Ah, sorry, the inputs there might be a bit confusing. The times are supposed to be in milliseconds, so you should enter much bigger numbers to have sufficient time.

**Comment by GM-ILLESCAS** [comment]
Dear Fabian:

Miguel told me that you would like to have the image of the original
dragon. I am attaching all the graphic supports we have, in case any of
them are useful to you. Anyway, I've seen the piece you designed for the
app and it's great!

https://we.tl/t-guPTpNtPRI

Cheers,
Carlos J. Penin - Ajedrez21

**Comment by GM-ILLESCAS** [comment]
Thanks Fabian!
I am already playing on pychess.com !! And I must say that the distance
between humans and Fairy Stockfish at dragonchess is even greater than in
normal chess.

I hope to organize a tournament very soon, I will let you know!

Miguel



[image: image.png]
